redash/models/__init__.py

```diff
@@ -256,7 +274,7 @@ def to_dict(self):
             'id': self.id,
             'query_hash': self.query_hash,
             'query': self.query_text,
-            'data': json_loads(self.data),
+            'data': self.data,
```

Comment:
We now deserialize it in a single location instead of having every user call `json_loads`.